crates/services/p2p/src/ports.rs

```diff
@@ -21,6 +22,11 @@ pub trait P2pDb: Send + Sync {
         height: &BlockHeight,
     ) -> StorageResult<Option<SealedBlockHeader>>;
 
+    fn get_sealed_headers_range(
```

Comment:
Can we consider simply calling this `get_sealed_headers`? I think `_range` is not idiomatic in the context of the rest of our interfaces, where we may already accept a range without putting that in the function name.

Reply:
Yeah. Totally. I'll fix it.